resources/scss/legacy-variables.scss

```diff
@@ -0,0 +1,8 @@
+$gray-lightest:             #f7f7f9 !default;
```

Comment:
Instead of what @dblaichinger suggested, have a look at [this guide](https://developers.plentymarkets.com/dev-doc/ceres-5#scss), which basically tells you which variables were replaced.
I am certain that I already replaced these variables once, but this change may got lost somehow. Anyway, the provided link should be an easy guide and after these changes, you should also be able to build SCSS again.